newsroom/companies/views.py

```diff
@@ -176,15 +175,16 @@ async def edit_company(args: CompanyItemArgs, params: None, request: Request) ->
 @company_endpoints.endpoint("/companies/<string:company_id>", methods=["DELETE"])
 @admin_only
 async def delete_company(args: CompanyItemArgs, params: None, request: Request) -> Response:
+    from newsroom.users.service import UsersService
+
     service = CompanyService()
     original = await service.find_by_id(args.company_id)
 
     if not original:
         raise NotFound(gettext("Company not found"))
 
     try:
-        # TODO-ASYNC: Convert to users async service
-        get_resource_service("users").delete_action(lookup={"company": args.company_id})
+        await UsersService().delete_many(lookup={"company": str(args.company_id)})
```

Comment:
It's probably due to the `Store ObjectId in MongoDB from async resource services` bug, but should this be an ObjectId when looking up which users to delete?

Reply:
yeah, I'll adjust it now that you prepared the fix.